app/views/images/index.html.haml

```diff
@@ -0,0 +1,31 @@
+%div{:class => "addimages"}
+  %a(href='images/new') Add an image
+%br
+%div{:class => "images"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace